### PR TITLE
RNs-4.8.52 Added with Bug Fix Update

### PR DESCRIPTION
**Pull request by Wiharris**
**RNs-4.8.52**
Added with Bug Fix Update

**Version(s):**
PR applies only to enterprise-4.8

**Issue:**
https://issues.redhat.com/browse/OSDOCS-4389

**Link to docs preview:**
https://wiharris.github.io/openshift-docs/RNs-4.8.52/release_notes/ocp-4-8-release-notes.html#ocp-4-8-52

QE review:
- [ ] QE has approved this change.

**Additional information:**
Erratas has `shipped live`

### CODE REVIEW

File: release_notes/ocp-4-8-release-notes.adoc [hunk 1]
```diff
@@ -3638,3 +3638,25 @@ $ oc adm release info 4.8.51 --pullspecs
...
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.8.52 --pullspecs
+----
+
+[id="ocp-4-8-52-bug-fixes"]
+==== Bug fixes
+* Previously, the liveness probe woud timeout after one second, causing the kuryr-controller to restart. This fix increases the default time limit, resulting in kuryr-controller running for a longer period. (link:https://issues.redhat.com/browse/OCPBUGS-472?jql=key%20in%20(%27OCPBUGS-472%27%2C%27OCPBUGS-1977%27%2C%27OCPBUGS-1023%27)[*OCPBUGS-216*])
```
Comment (GroceryBoyJr) [inline release_notes/ocp-4-8-release-notes.adoc:3658]: I'd add the missing lowercase-L to "woud"

**Review by GroceryBoyJr — COMMENTED** [review]
William - just the one small nit, otherwise LGTM!

File: release_notes/ocp-4-8-release-notes.adoc (hunk 1 above)
Comment (Wiharris) [reply release_notes/ocp-4-8-release-notes.adoc]: Thanks for catching, James! I have made the updates! :)

**Review by bergerhoffer — COMMENTED** [review]
LGTM merging!

**Pull request merged**